obsolete/led-rgb-4pin-cathode_rbg_v4.fzp

```diff
@@ -12,7 +12,7 @@
     </tags>
     <properties>
         <property name="package">5 mm [THT]</property>
-        <property name="family">LED</property>
+        <property name="family">//obsolete//LED</property>
```

Comment:
Which change required the obsoletion?

Reply:
Ah yes, completely valid. The ID needs to be changed, and it doesn't make sense to have two versions of the part, so the one with the previous ID gets replaced.